src/applications/vaos/tests/e2e/workflows/request-schedule-workflow/mental-health.cypress.spec.js

```diff
@@ -0,0 +1,165 @@
+  APPOINTMENT_STATUS,
+  TYPE_OF_CARE_IDS,
+} from '../../../../utils/constants';
+import MockAppointmentResponse from '../../../fixtures/MockAppointmentResponse';
+import MockClinicResponse from '../../../fixtures/MockClinicResponse';
+import MockEligibilityResponse from '../../../fixtures/MockEligibilityResponse';
+import MockFacilityResponse from '../../../fixtures/MockFacilityResponse';
+import MockSlotResponse from '../../../fixtures/MockSlotResponse';
+import MockUser from '../../../fixtures/MockUser';
+import AppointmentListPageObject from '../../page-objects/AppointmentList/AppointmentListPageObject';
+import ConfirmationPageObject from '../../page-objects/ConfirmationPageObject';
+import ContactInfoPageObject from '../../page-objects/ContactInfoPageObject';
+import DateTimeRequestPageObject from '../../page-objects/DateTimeRequestPageObject';
+import ReasonForAppointmentPageObject from '../../page-objects/ReasonForAppointmentPageObject';
+import ReviewPageObject from '../../page-objects/ReviewPageObject';
+import TypeOfCarePageObject from '../../page-objects/TypeOfCarePageObject';
+import VAFacilityPageObject from '../../page-objects/VAFacilityPageObject';
+import {
+  mockAppointmentCreateApi,
+  mockAppointmentGetApi,
+  mockAppointmentsGetApi,
+  mockClinicsApi,
+  mockEligibilityApi,
+  mockFacilitiesApi,
+  mockFeatureToggles,
+  mockSchedulingConfigurationApi,
+  mockSlotsApi,
+  mockVamcEhrApi,
+  vaosSetup,
+} from '../../vaos-cypress-helpers';
+import TypeOfVisitPageObject from '../../page-objects/TypeOfVisitPageObject';
+
+const { idV2: typeOfCareId } = getTypeOfCareById(
+  TYPE_OF_CARE_IDS.MENTAL_HEALTH,
+);
+const typeOfCareRegex = /Mental health/i;
+
+describe('VAOS direct schedule flow - Mental health', () => {
+  describe('When patient has no history and flipper for MH past filtering is enabled', () => {
+    beforeEach(() => {
+      vaosSetup();
+
+      const response = new MockAppointmentResponse({
+        id: 'mock1',
+        localStartTime: new Date(),
+        status: APPOINTMENT_STATUS.proposed,
+        pending: true,
+      }).setType('REQUEST');
+      mockAppointmentGetApi({ response });
+      mockAppointmentCreateApi({ response });
+      mockAppointmentsGetApi({ response: [] });
+      mockFeatureToggles();
+      mockVamcEhrApi();
+    });
+
+    describe('And one facility supports direct scheduling with no history required', () => {
+      const setup = () => {
+        const mockEligibilityResponse = new MockEligibilityResponse({
+          facilityId: '983',
+          typeOfCareId,
+          isEligible: true,
+        });
+
+        mockFacilitiesApi({
+          response: MockFacilityResponse.createResponses({
+            facilityIds: ['983'],
+          }),
+        });
+        mockEligibilityApi({ response: mockEligibilityResponse });
+        mockSchedulingConfigurationApi({
+          facilityIds: ['983'],
+          typeOfCareId,
+          isDirect: false,
+          isRequest: true,
```

Comment:
Hmmm I think this configuration is incorrect. What we want to test here is if both direct and request are enabled, and direct requires patient history, but the patient does not have history, then it falls back to the request flow.

Reply:
Ah, got it, but should that be a separate test with the fallback and/or disabled?

Reply:
I made both true. Ah, I found where I saw it false for one and true for the other in: `src/applications/vaos/tests/e2e/workflows/request-schedule-workflow/primary-care.cypress.spec.js`